portfolio/src/main/webapp/about.js

```diff
@@ -0,0 +1,275 @@
+function createMap() {
+  const map = new google.maps.Map(
+      document.getElementById('map'),
+      {center: {lat: 37.422403, lng: -122.088073}, zoom: 15});
+ 
+  addLandmark(
+      map, 37.423829, -122.092154, 'Google West Campus',
+      'Google West Campus is home to YouTube and Maps.')
+  addLandmark(
+      map, 37.421903, -122.084674, 'Stan the T-Rex',
+      'This is Stan, the T-Rex statue.')
+  addLandmark(
+      map, 37.420919, -122.086619, 'Permanente Creek Trail',
+      'Permanente Creek Trail connects Google to a system of bike trails.');
+}
+ 
+/** Adds a marker that shows an info window when clicked. */
+function addLandmark(map, lat, lng, title, description) {
+  const marker = new google.maps.Marker(
+      {position: {lat: lat, lng: lng}, map: map, title: title});
+ 
+  const infoWindow = new google.maps.InfoWindow({content: description});
+  marker.addListener('click', () => {
+    infoWindow.open(map, marker);
+  });
+}
+
+
+
+
+/*
+// Create greyscale map with a marker 
+function initMap() {
+    // Create a new StyledMapType object, passing it an array of styles,
+    // and the name to be displayed on the map type control.
+    var styledMapType = new google.maps.StyledMapType(
+    [{
+        "elementType": "geometry",
+        "stylers": [
+        {
+            "color": "#f5f5f5"
+        }
+        ]
+    },
+    {
+        "elementType": "labels.icon",
+        "stylers": [
+        {
+            "visibility": "off"
+        }
+        ]
+    },
+    {
+        "elementType": "labels.text.fill",
+        "stylers": [
+        {
+            "color": "#616161"
+        }
+        ]
+    },
+    {
+        "elementType": "labels.text.stroke",
+        "stylers": [
```

Comment:
I think this giant commented out region can also be deleted, right?

Reply:
there is actually a bug with the map display after trying to reconfigure my portfolio, i am saving it to back to this weekend